pipeline/nextflow_slurm_custom.config

```diff
@@ -0,0 +1,164 @@
+params.executor = "slurm"
+
+// define your preferred queues here
+params.default_queue = 'mit_normal_gpu'  // Default queue
+params.gpu_queue = null  // Optional: Only set if GPU queue is needed and different from default queue
```

Comment:
anything below this line specific to MIT?  I just wonder how/if we could modularize ...

Reply:
The entire file is specifically tailored to the usage at MIT, which differs slightly from their general templates. Their templates are very general suggestions and I would not overthink how to 'modularize' to all cases